### PR TITLE
Setup for Amazon S3 Bucket

### PR DESCRIPTION
**Pull request by g4rry420**
### What this PR does (required):
- Setup for the Amazon S3 bucket  to upload images #35 

### Screenshots / Videos (required):
N/A

### Any information needed to test this feature (required):
- Use Postman to test this feature currently.

### Any issues with the current functionality (optional):
No

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -43,6 +44,7 @@ app.use((req, res, next) => {
 
 app.use("/auth", authRouter);
 app.use("/users", userRouter);
+app.use("/s3", s3Router);
```
Comment (dproc96) [inline server/app.js]: Would be clearer to have this route be something like `/upload`

File: server/controllers/s3.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    Bucket: process.env.BUCKET_NAME,
+    Body: req.file.buffer,
+    ContentType: req.file.mimetype,
+    Key: `photo/${req.file.originalname}`
+  };
+
+  s3.upload(params, (error, data) => {
+    if (error) {
+      return res.status(400).json({
+        error
```
Comment (dproc96) [inline server/controllers/s3.js]: I think it would be better to log this error on the server and pass a more generic error to the user

**Review by dproc96 — APPROVED** [review]
Looks good, two small change requests but once you make those, this is good to merge

File: server/app.js (hunk 1 above)
Comment (g4rry420) [reply server/app.js]: Sure, Will do that

File: server/controllers/s3.js (hunk 2 above)
Comment (g4rry420) [reply server/controllers/s3.js]: Will do that, thanks

**Pull request merged**